bin/nodeshift

```diff
@@ -75,17 +76,33 @@ yargs
     type: 'boolean',
     default: false
   })
+  .options('metadata.out', {
+    describe: 'determines what should be done with the response metadata from OpenShift',
+    choices: ['stdout', 'ignore', '<filename>'],
+    type: 'string',
+    default: 'ignore'
+  })
   .help().argv;
 
 function commandHandler (argv) {
   const options = createOptions(argv);
-  require('./cli')(options).then(logger.info).catch((err) => {
-    const log = require('../lib/common-log')();
-    log.error(err.message);
-    log.error('Status code', err.statusCode);
-    require('util').debuglog('nodeshift.cli')(err.stack);
-    process.exit(1);
-  });
+  require('./cli')(options).then((response) => {
+    if (options.metadata) {
+      if (options.metadata.out === 'stdout') {
+        process.stdout.write(JSON.stringify(response, null, 2));
+      } else {
+        fs.writeFileSync(options.metadata.out, JSON.stringify(response, null, 2), 'utf8');
```

Comment:
i actually used a module called jsonfile here https://github.com/bucharest-gold/nodeshift/blob/master/lib/resource-writer.js#L26 to output to json,   but i like this approach better.  looks like i'll update that other part so we can get rid of a dependecy